### ISSUE TITLE
Neutral label

### ISSUE THREAD

**Issue by chaoyu12345** [donydchen/FMPN-FER#6]
Hi, I wonder how to deal with the training data with neutral label.

**Comment by donydchen** [comment]
Sorry but I was busy in the past few days. 
As per the paper, this project _only_ focuses on those expressive faces, following general settings of other related SOTA works. 
For neutral face images, they are only leveraged to build pseudo masks in the data preprocess phase. That is, neutral faces are not directly used in training / testing. 
Hope it help.
